packages/admin/src/emailCampaigns/EmailCampaignsGrid.tsx

```diff
@@ -202,6 +202,11 @@ export function EmailCampaignsGrid({
                             }
                             refetchQueries={[emailCampaignsQuery]}
                         />
+                        {row.sendingState === "SENT" && (
+                            <IconButton component={StackLink} pageName="statistics" payload={row.id}>
+                                <Statistics color="primary" />
+                            </IconButton>
+                        )}
```

Comment:
Shouldn't this be shown before the context menu? 
Usually the three dots of the context menu are shown at the end.

Reply:
okay 👍 
changed in: 581a8d199439708fcf2363e7f4039a6310de2c34

looks like that now: 
<img width="1398" alt="Screenshot 2024-03-19 at 12 52 20" src="https://github.com/vivid-planet/comet-brevo-module/assets/44967610/cabde893-8002-4dd6-8f5b-10aee4d373a0">